stubs/html5lib/METADATA.toml

```diff
@@ -1 +1,4 @@
 version = "1.1.*"
+
+[tool.stubtest]
+extras_dependencies = ["genshi"]
```

Comment:
Why not also the `lxml` extra? (Or even just `all`)

Reply:
LXML is not available as an extras on the versions of Python being tested, as I understand it:

https://github.com/html5lib/html5lib-python/blob/master/setup.py#L114

Reply:
To clarify: if lxml is no longer required to perform testing, I can remove it. It simply wasn't clear that was the case. I was ensuring that the installed extras would be equivalent to the existing test runtimes.

Reply:
I had misread the setup.py as `Cython` instead of `CPython`, so I've updated this to all in 4cf8bf8. Apologies for that